### PR TITLE
Doc/e2e

### PR DESCRIPTION
**Pull request by SuttArt**
### Documentation Changes
- [x] add `create-e2e-test.md` documentation

### CODE REVIEW

File: doc/compodoc_sources/how-to-guides/add-cypress-to-project.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# How to add Cypress to project
```
Comment (TheSlimvReal) [inline doc/compodoc_sources/how-to-guides/add-cypress-to-project.md:1]: As no-one of the project will have to setup Cypress again I would remove this small guide and move the `How to use Cypress` to the other guide. To have it all in one place and don't clutter up the documentation with unnecessary guides.

**Review by TheSlimvReal — CHANGES_REQUESTED** [review]
Thanks for creating the PR. I can follow quite well but I think there are some useful things to add:

- [x] How does Gherkin help with writing Cypress tests
- [ ] How to use the Cypress interface
- [ ] Small tricks that are relevant especially for writing tests for Aam Digital

For each of them you can either write your own section or just link to a good guide.

File: doc/compodoc_sources/how-to-guides/create-e2e-test.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## Resources and How-Tos
+* [Documentation](https://docs.cypress.io/guides/getting-started/writing-your-first-test)
+* [Introduction to Cypress](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress)
+* [Writing and Organizing Tests](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests)
+
+## Instruction
+### First Step
+#### Create a file
+All tests are located under `ndb-core/e2e/integration` directory. Good practice is to create for every single test his own directory. The name of test and directory should have the same name.
```
[the remark this answers is not in the document]
Comment (SuttArt) [reply doc/compodoc_sources/how-to-guides/create-e2e-test.md:11]: You're right, corrected.

File: doc/compodoc_sources/how-to-guides/create-e2e-test.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+```
+### Third Step
+#### Implement E2E-Test
+```
+describe("Scenario of the Test", () => {
+  before(() => {
+  // In the before() function we can describe our start Point and specific variables
+  //e.g.
+    cy.visit("http://localhost:4200");
+    cy.wrap("Something").as("VariableName");
```
[the remark this answers is not in the document]
Comment (SuttArt) [reply doc/compodoc_sources/how-to-guides/create-e2e-test.md:36]: Done

File: doc/compodoc_sources/how-to-guides/add-cypress-to-project.md (hunk 1 above)
Comment (SuttArt) [reply doc/compodoc_sources/how-to-guides/add-cypress-to-project.md:1]: Done

File: doc/compodoc_sources/how-to-guides/create-e2e-test.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+<br />
+e.g.
+####Scenario: Linking a child to a school
+```
+Given I am on the details page of a child
+When I add an entry in the 'Previous Schools' section with a specific school
+Then I can see that child in the 'Children Overview' of the details page of this school
+```
+### Third Step
+#### Implement E2E-Test
```
[the remark this answers is not in the document]
Comment (SuttArt) [reply doc/compodoc_sources/how-to-guides/create-e2e-test.md:83]: We need to discuss this

File: doc/compodoc_sources/how-to-guides/create-e2e-test.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+});
+
+```
+[A solid test generally covers 3 phases](https://docs.cypress.io/guides/getting-started/writing-your-first-test#Write-a-real-test):
+
+<ol>
+<li>Set up the application state.</li>
+<li>Take an action.</li>
+<li>Make an assertion about the resulting application state.</li>
+</ol>
```
[the remark this answers is not in the document]
Comment (SuttArt) [reply doc/compodoc_sources/how-to-guides/create-e2e-test.md:122]: We need to discuss this too.

File: doc/compodoc_sources/how-to-guides/create-e2e-test.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (SuttArt) [reply doc/compodoc_sources/how-to-guides/create-e2e-test.md:83]: Done

**Review by TheSlimvReal — APPROVED** [review]
Thanks for finalizing this!

**Pull request merged**